### PR TITLE
Linted

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

**Comment by tannewt** [comment]
Depends on https://github.com/adafruit/Adafruit_CircuitPython_USB_Host_Descriptors/pull/1

**Review by jepler — APPROVED** [review]
the computer eats the lint so we don't have to

File: adafruit_usb_host_mass_storage.py [hunk 1]
```diff
@@ -32,27 +35,39 @@
...
+_SCSI_CMD_READ_10 = const(0x28)
+"""The READ (10) command requests that the device server read the specified logical block(s) and
+transfer them to the data-in buffer."""
+_SCSI_CMD_WRITE_10 = const(0x2A)
+"""The WRITE (10) command requests that the device server transfer the specified logical block(s)
+from the data-out buffer and write them."""
+
 
 class USBMassStorage:
+    """CircuitPython BlockDevice backed by a USB mass storage device (aka thumb drive.)"""
```
Comment (dhalbert) [inline adafruit_usb_host_mass_storage.py]:
```suggestion
    """CircuitPython BlockDevice backed by a USB mass storage device (aka thumb drive)."""
```

File: README.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ Dependencies
 =============
 This driver depends on:
 
-* `Adafruit CircuitPython <https://github.com/adafruit/circuitpython>`_
+* `Adafruit CircuitPython 9+ <https://github.com/adafruit/circuitpython>`_
```
Comment (dhalbert) [inline README.rst]:
```suggestion
* `Adafruit CircuitPython 9.0.0 and later <https://github.com/adafruit/circuitpython>`_
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Looks fine -- just some typos.

File: README.rst (hunk 2 above)
Comment (dhalbert) [reply README.rst]: Aha, do you mean 9.0.0? I get it. Maybe say `9.0.0 and later`

**Comment by tannewt** [comment]
Still waiting for you @dhalbert

**Comment by dhalbert** [comment]
> Still waiting for you @dhalbert

Sorry! I did not see the re-review request.

**Pull request merged**